.github/workflows/test-browserstack.yml

```
@@ -41,6 +41,14 @@ jobs:
     name: Run Browserstack
     runs-on: ubuntu-latest
     needs: [ build_core ]
+    # The concurrency field allows us to block multiple invocations of this job across multiple workflow runs.
+    # Stencil is only able to run 5 parallel tests in Browserstack, which is exceeded when more than one run of this
+    # workflow occurs - e.g. one or more pull requests submitted in proximity, a pull request and a merge to a branch
```

note to self:
```suggestion
    # workflow occurs - e.g. more than one pull request is submitted in proximity, a pull request and a merge to a branch
```

do this in the editor, double check for exceeding 120 chars